cli/lib/kontena/client.rb

```diff
@@ -332,13 +332,10 @@ def request(http_method: :get, path:'/', body: nil, query: {}, headers: {}, resp
         retry if refresh_token
       end
       raise Kontena::Errors::StandardError.new(401, 'Unauthorized')
-    rescue Excon::Errors::NotFound
-      raise Kontena::Errors::StandardError.new(404, 'Not found')
-    rescue Excon::Errors::Forbidden
-      raise Kontena::Errors::StandardError.new(403, 'Access denied')
-    rescue
-      logger.debug "Request exception: #{$!} - #{$!.message}\n#{$!.backtrace.join("\n")}"
-      handle_error_response
+    rescue Excon::Errors::HTTPStatusError => error
+      logger.debug "Request #{error.request[:method].upcase} #{error.request[:path]}: #{error.response.status} #{error.response.reason_phrase}: #{error.response.body}"
+
```

Comment:
`@last_response` is a bad practice because it keeps the state and because of that it will have a wrong value if first request is successful and next one throws an error.